### DISCUSSION TITLE
Why the cursor property on a button is default?

### DISCUSSION THREAD

**Discussion by elVengador** [adobe/react-spectrum#5787] [Q&A]
I noticed that the cursor property on a button is set to default. What is its purpose? I checked buttons on other websites, and they all have `cursor: pointer` applied.

https://github.com/adobe/react-spectrum/assets/27080940/e0a31dea-c372-4516-918f-384983cec5e2

**Comment by reidbarber** [comment]
Check out this section of the FAQ https://github.com/adobe/react-spectrum/wiki/Frequently-Asked-Questions-(FAQs)#why-dont-buttons-use-a-pointer-cursor

Also, Spectrum guidelines specify that the default cursor should be used for buttons, unless they are semantically links: https://spectrum.adobe.com/page/button/#Cursor-direction

**Reply by steveoh** [reply]
I read that wiki and looked at microsoft, adobe, and apples websites and it doesn't seem that this guideline is being followed. I'm thinking I must be misinterpreting the guidance. Can you clarify?

On adobe's website... these are not semantic links yet they use the pointer.
<img width="715" alt="image" src="https://github.com/user-attachments/assets/8648d3c5-08a1-41ac-9fa9-da0107b13906">

Apple does the same with summary descriptions and the customize the product buttons all use pointers.
<img width="359" alt="image" src="https://github.com/user-attachments/assets/b53263d1-f879-4cc7-9eb3-5f8d7d407c33">

This microsoft carousel control uses pointers for the pause play and forward back. Are these semantic links? They don't seem like it to me.
<img width="239" alt="image" src="https://github.com/user-attachments/assets/5ba49056-4ccd-41bb-a096-2b1eeea6f79c">

**Reply by reidbarber** [reply]
Yeah, I'm not sure every team/product follows these guidelines perfectly, but we wanted to as a component library.
